osu.Game/Graphics/UserInterface/OsuModSettingsTextBox.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics.Colour;
+using osu.Framework.Input.Events;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public class OsuModSettingsTextBox : OsuTextBox
+    {
+        private const float border_thickness = 3;
+
+        private SRGBColour borderColourFocused;
+        private SRGBColour borderColourUnfocused;
```

Comment:
Not sure why these are using `SRGBColour` really, should really be `Colour4` I think. Using the SRGB type directly here just adds confusion about colour space stuff. The framework provides implicit conversions that handle the switch just fine.